docs/reference/display.md

```diff
@@ -703,7 +704,7 @@ They should be deleted with the `wb_display_image_delete` function when they are
 Finally, note that both the main display image and the clipboard images have an alpha channel.
 
 The `wb_display_image_new` function creates a new clipboard image, with the specified `with` and `height`, and loads the image `data` into it with respect to the defined image `format`.
-Three images format are supported: `WB_IMAGE_RGBA` which is similar to the image format returned by a `Camera` device and `WB_IMAGE_RGB` or `WB_IMAGE_ARGB`.
+Five images format are supported: `WB_IMAGE_BGRA` which is similar to the image format returned by a `Camera` device and `WB_IMAGE_RGB`, `WB_IMAGE_RGBA`, `WB_IMAGE_ARGB`,  or `WB_IMAGE_ABGR`.
 `WB_IMAGE_RGBA` and `WB_IMAGE_ARGB` are including an alpha channel respectively after and before the color components.
```

Comment:
But also `WB_IMAGE_BGRA` and `WB_IMAGE_ABGR` (maybe we can remove this sentence which is not very helpful).
```suggestion
```